examples/ChordProg.hs

```diff
@@ -0,0 +1,69 @@
+{-# LANGUAGE TypeApplications #-}
+import Music.Prelude hiding (score)
+import Data.Traversable
+import qualified Data.List
+import Control.Monad.Bayes.Class
+import Control.Monad.Bayes.Enumerator (normalForm)
+
+main = defaultMain music
+
+music = inspectableToMusic @[Voice Pitch] $
+  [ view voice $ fmap fromPitch $ fst $ last $
+    Data.List.sortOn snd $ normalForm $ species1 c cf
+  , view voice $ fmap fromPitch $ cf
+  ]
+  where
+    cf =
+      [c,d,e,c,d]
```

Comment:
TODO: using Enumerator/normalForm this only works for a Cantus Firmus of up to 5 notes (6 is really slow). Use a better instance for `MonadInfer` which still allows drawing a high-scored sample (if not the highest).